src/lib/edra/headless/editor.ts

```
@@ -40,12 +50,37 @@ export interface EdraEditorProps {
 	) => Promise<void>;
+	/**
+	 * Called when the user clicks a math node. Forwards TipTap Mathematics `onClick`
+	 * for both block and inline math, and includes `isBlock` plus the editor instance.
+	 */
+	onMathClick?: MathClickHandler;
 }
 
-export const createEditor = (props?: EdraEditorProps) =>
-	useEditor({
+export const createEditor = (props?: EdraEditorProps) => {
+	let editorRef: Editor | undefined;
```

ESLint

[error] 'editorRef' is never reassigned. Use 'const' instead.

(prefer-const)